app/controllers/api/v1/players_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::V1::PlayersController < ApplicationController
+  def update
+    player = Player.find(params[:id])
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unscoped call to Player#find](https://www.codacy.com/app/Got-Board-Team/got-board-api/file/2382753688/issues/source?bid=3216569&fileBranchId=3216571#l3)